.gitignore

```diff
@@ -1,3 +1,9 @@
+# Contributer ignore ... where all local testing / learning stuff should be located
+# in this directory ... to not be included in pushes / merges / pull requests
+.local
+# to not include vscode configfolder in pushes / merges / pull requests
+.vscode
+# Ignore from official maintainer of repo
 INSTALL
```

Comment:
i mentioned in initial response - before commenting on specific code pieces - that we don't stuff unrelated changes into commits. maybe you overlooked that comment ?

Reply:
@rofl0r 
I had read it ... but i hoped it could evolve to an agreement, that we can ignore these 2 directories in general ;-)

VSCode (My IDE) simply all the time creates it in this location when i create project specific debug configurations.
And if i would remove it, then i need to configure it every time again when i visit this repo and want to work on it.

It simply creates much inconvinience ... if this entry could not be included in the global root .gitignore

The second directory exclude is for everything else which i store locally in this repo which is related to tinyproxy, notes, test scripts and so on.

This directory name i have choosen by intention.... that also other contributors could use such a location on there side and get the same benefit, that tinyproxy related stuff can be locally stored in the repo clone ... and it automaticall will be ignored by all pushes and merges / pull requests.

How you handle this problem in your side ?

Reply:
you can modify .gitignore locally without checking it in. just don't git add it.

Reply:
Understood .... done. 
.. and i had hoped ...